.dockerignore

```diff
@@ -0,0 +1 @@
+**/*.pyc
```

Comment:
I think this could also include `architect.yml`, `README.md`, `LICENSE.rst`, and `MANIFEST.in`